src/main/kotlin/fi/metatavu/noheva/devices/ExhibitionDeviceGroupController.kt

```diff
@@ -147,12 +143,14 @@ class ExhibitionDeviceGroupController {
    * Copies device group including all it's contents (devices, content versions, group content versions and pages).
    *
    * @param sourceDeviceGroup source device group
+   * @param targetContentVersionExhibition if not empty then new content versions should be assigned to this exhibition
    * @param creatorId id of user that created the copy
    * @return copied device group
    */
   fun copyDeviceGroup(
     idMapper: IdMapper,
     sourceDeviceGroup: ExhibitionDeviceGroup,
+    targetContentVersionExhibition: Exhibition?,
```

Comment:
remove this, targetExhibition should be derived from targetRoom (as target exhibition)